rte_mis.install

```diff
@@ -20,6 +21,32 @@ function rte_mis_install() {
   $config = \Drupal::configFactory()->getEditable('administerusersbyrole.settings');
   $roles = $config->get('roles') ?: [];
   $roles['block_admin'] = 'safe';
+  $roles['district_admin'] = 'safe';
+  $roles['school'] = 'perm';
+  $roles['school_admin'] = 'perm';
   $config->set('roles', $roles)->save();
 
+  $role_id = 'state_admin';
+  $permissions = [
+    'view users with role school',
+    'view users with role school_admin',
+    'edit users with role school',
+    'edit users with role school_admin',
+  ];
+
+  // Load the role entity.
+  $role = Role::load($role_id);
+
+  if ($role) {
+    // Grant each permission to the role.
+    foreach ($permissions as $permission) {
```

Comment:
Check the issue - https://www.drupal.org/project/administerusersbyrole/issues/3308308

Reply:
What about this isssue?

Reply:
What about this patch?

Reply:
I have gone through this issue. Also applied the latest patch but still this is working. Hence, we had to do this.

Reply:
Use this patch - https://www.drupal.org/project/drupal/issues/3358586#comment-15624429, 

Permission that we are updating in install file can be added in the config file(state role, administer settings). This patch would bypass the fatal error and allow the site to be installed.